Std/Data/Nat/Lemmas.lean

```diff
@@ -1145,6 +1145,21 @@ protected theorem mul_pow (a b n : Nat) : (a * b) ^ n = a ^ n * b ^ n := by
   | zero => rw [Nat.pow_zero, Nat.pow_zero, Nat.pow_zero, Nat.mul_one]
   | succ _ ih => rw [Nat.pow_succ, Nat.pow_succ, Nat.pow_succ, Nat.mul_mul_mul_comm, ih]
 
+protected theorem one_lt_two_pow (h : n ≠ 0) : 1 < 2 ^ n :=
+  match n, h with
+  | n+1, _ => by
+    rw [Nat.pow_succ', ← Nat.one_mul 1]
+    exact Nat.mul_lt_mul_of_lt_of_le' (by decide) (Nat.pow_two_pos n) (by decide)
```

Comment:
How about make this a simp rule?  `1 < 2 ^ n ↔ 0 < n`?  Then `one_lt_two_pow_succ` probably won't beneeded?

Reply:
I added the simp lemma, and removed one_lt_two_pow_succ